public/app/core/components/AppChrome/MegaMenu/MegaMenuHeader.tsx

```diff
@@ -27,17 +25,26 @@ export function MegaMenuHeader({ handleMegaMenu, handleDockedMenu, onClose }: Pr
   return (
     <div className={styles.header}>
       <Stack alignItems="center" minWidth={0} gap={0.25}>
-        <ToolbarButton
+        { // begin NI fork changes
+        <IconButton
+            onClick={handleMegaMenu}
+            tooltip={t('navigation.megamenu.close', 'Close menu')}
+            name='arrow-from-right'
+            style={{ transform: 'rotate(180deg)' }}
+          >
+        </IconButton>
+        /* <ToolbarButton
           narrow
           id={MEGA_MENU_HEADER_TOGGLE_ID}
           onClick={handleMegaMenu}
           tooltip={t('navigation.megamenu.close', 'Close menu')}
-        >
+                  >
           <Branding.MenuLogo className={styles.img} />
         </ToolbarButton>
-        <OrganizationSwitcher />
+        <OrganizationSwitcher /> */
+        }
       </Stack>
-      <IconButton
+      {/* <IconButton
```

Comment:
This removes the button responsible for docking the menu, which we have decided we don't want.